install/src/main/java/org/jetbrains/bsp/bazel/install/ProjectViewCLiOptionsProvider.kt

```diff
@@ -45,6 +47,16 @@ object ProjectViewCLiOptionsProvider {
         return ProjectViewTargetsSection(includedTargets, excludedTargets)
     }
 
+    private fun toDirectoriesSection(projectViewCliOptions: ProjectViewCliOptions?): ProjectViewDirectoriesSection? =
+            projectViewCliOptions?.directories?.let(::toDirectoriesSectionNotNull)
+
+    private fun toDirectoriesSectionNotNull(directories: List<String>?): ProjectViewDirectoriesSection {
```

Comment:
it can be just `directories: List<String>` - and u can fix it in `toTargetsSectionNotNull` as well

Reply:
Fixed